spec/system/request_for_comments_filter_system_spec.rb

```diff
@@ -37,4 +44,19 @@
     visit(request_for_comments_path)
     expect(page).to have_css('ul.pagination')
   end
+
+  context 'when reporting is enabled' do
```

Comment:
Could you please also test the "unhappy" path? --> "when reporting is disabled" :pray:

Reply:
OK. Removed.